integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/specs/TasklistSpecification.groovy

```diff
@@ -71,7 +71,7 @@ class TasklistSpecification extends GebReportingSpec {
     when: 'I go to the tasklist page with all sections complete'
     db.createData(12, JsonOutput.toJson([
       ratings: [
-        offendingHistory: [previousConvictions: "some convictions"],
+        offendingHistory: [previousConvictions: "Yes", previousConvictionsText: "some convictions", furtherCharges: "No"],
```

Comment:
I think I might extract some of this common db setup into one place in a future commit